pom.xml

```diff
@@ -82,6 +80,79 @@
                     <target>${java.version}</target>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-site-plugin</artifactId>
+                <version>3.8.2</version>
+                <configuration>
+                    <locales>en</locales>
+                </configuration>
+            </plugin>
         </plugins>
+
     </build>
+
+     <scm>
```

Comment:
The scm field also lacks a webBrowser view field
<img width="549" alt="Screenshot 2020-01-20 at 16 09 51" src="https://user-images.githubusercontent.com/45944760/72737161-54fe1d00-3b9f-11ea-9877-7804da130122.png">


Also : the links are in ssh but ssh access is authorized only for dev in the ARA team of Decathlon's. Maybe we should display the https link instead to force user do Fork & PR ? If we keep the ssh links : the links are malformated, good one is `git@github.com:Decathlon/ara.git`

Reply:
updated !